main.cpp

```diff
@@ -127,393 +163,337 @@ struct CarWash
 };
-/*
-Thing 8) Tuners
-5 properties:
-    1) tuner metal (std::string)
-    2) string tension in pounds (double)
-    3) number of tuners(int)
-    4) tuner angle in degrees (double)
-    5) tuner gear ratio (float)
-3 things it can do:
-    1) rotate tuner key
-    2) adjust string tension
-    3) tighten set nut
- */ 
+
+int Body::selectPickup(int pickup) { return pickup;}
+
+float Body::changeVolume(float newVolume) { return newVolume;}
+
+float Body::changeTone(float newTone){ return newTone;}
+
 struct Tuners
 {
-    // tuner metal 
     std::string metal = "Steel";
-    // adjustment key orientation 
     double stringTension = 11.8;
-    // number of tuners
     int numTuners = 6;
-    // tuner angle in degrees
     double degree = 1.75;
-    // tuner gear ratio 
-    float gearRatio = 50.0;
+    double gearRatio = 50.0;
 
-    // rotate tuner key
-    void rotateKey(int keyNum = 1, float numTurns = 0.f);
-    // adjust string tension
+    void rotateKey(int keyNum = 1, double numTurns = 0.0);
     void changeStringTension(int keyNum, int tunedNote);
-    // get number of installed strings
     int getNumInstalledStrings();
     
 };
-/*
-Thing 9) Bridge
-5 properties:
-    1) type of bridge(std::string)
-    2) number of string poles (int)
-    3) length adjustment screw setting in mm (double)
-    4) type of tremolo (std::string)
-    5) floating bridge tightness setting (std::string)
-3 things it can do:
-    1) adjust string length
-    2) move whammy bar
-    3) adjust bridge height
- */
+
+void Tuners::rotateKey(int keyNum , double numTurns) 
+{
+    degree += keyNum*numTurns/gearRatio*360.0;
+}
+
+void Tuners::changeStringTension(int keyNum, int tunedNote)
+{
+    stringTension += keyNum*tunedNote*0.01;
+}
+
+int Tuners::getNumInstalledStrings() {return numTuners;}
+
 struct Bridge
 {
-    // type of bridge
     std::string type = "floating";
-    // number of string poles 
     int numStrings = 6;
-    // length adjustment screw setting in mm 
     double screwSetting = 0.5;
-    // type of tremolo 
     std::string tremoloType = "strat";
-    // floating bridge tension (std::string)
     std::string tension = "high";
 
-
-    // adjust string length
     void adjustScrewSetting(double screwAdjustment);
-    // move whammy bar
     void rotateWhammy(std::string direction);
-    // adjust bridge height
     void changeHeight(double heightAdjustment);
 };
-/*
-Thing 10) Electric Guitar
-5 properties:
-    1) Pickups
-    2) Neck
-    3) Body
-    4) Tuners
-    5) Bridge
-3 things it can do:
-    1) play a chord
-    2) generate electrical output
-    3) tune strings
- */
+
+void Bridge::adjustScrewSetting(double screwAdjustment) 
+    { screwSetting += screwAdjustment;}
+
+void Bridge::rotateWhammy(std::string direction) 
+{
+    if (direction == "pull") tension = "high";
+    else                     tension = "low";
+}
+
+void Bridge::changeHeight(double heightAdjustment) { screwSetting += heightAdjustment;}
```

Comment:
If you write single-line functions, make sure you have space around the `{ }` so the code is easier to read
- [ ] revise